packages/react-sdk/src/components/PendingCallPanel/PendingCallPanel.tsx

```diff
@@ -0,0 +1,53 @@
+import { CallingState } from '@stream-io/video-client';
+import {
+  useCall,
+  useCallCallingState,
+  useI18n,
+} from '@stream-io/video-react-bindings';
+import { Avatar } from '../Avatar';
+import { PendingCallControls } from './PendingCallControls';
+
+export const PendingCallPanel = () => {
+  const call = useCall();
+  const callingState = useCallCallingState();
+  const { t } = useI18n();
+
+  if (!call) return null;
+
+  const caller = call.data?.created_by;
+  const membersToShow = call.isCreatedByMe
+    ? call.state.members
+        ?.slice(0, 3)
+        .map(({ user }) => user)
+        .filter((u) => !!u) || []
+    : caller
+    ? [caller]
+    : [];
+
+  return (
+    <div className="str-video__call-panel str-video__call-panel--pending">
+      <div className="str-video__call-panel__members-list">
+        {membersToShow.map((user) => (
+          <div key={user.id} className="str-video__call-panel__member-box">
+            <Avatar name={user.name} imageSrc={user.image} />
+          </div>
+        ))}
+      </div>
+      {callingState === CallingState.RINGING && <div>{t('Ringing')}</div>}
+      {callingState === CallingState.JOINING && <div>{t('Joining')}</div>}
+      {callingState === CallingState.RECONNECTING && (
+        <div>{t('Re-connecting')}</div>
+      )}
+      {callingState === CallingState.RECONNECTING_FAILED && (
+        <div>{t('Failed')}</div>
+      )}
+      {callingState === CallingState.OFFLINE && (
+        <div>{t('No internet connection')}</div>
+      )}
```

Comment:
I guess these `<div>` elements should have a class name attached to them, in case someone wants to apply some styles to them.